frontend/pages/kickae/[id].tsx

```diff
@@ -0,0 +1,145 @@
+import styles from '/styles/kickae.module.scss'
+import Image from 'next/image'
+import { useEffect, useLayoutEffect, useState } from 'react'
+import TeamCard from '../../components/TeamCard';
+import GoalDisplay from '../../components/GoalDisplay';
+
+import {QRCodeSVG} from 'qrcode.react';
+import { useRouter } from 'next/router'
+
+export default function KickaePage() {
+    const router = useRouter()
+
+    const [scoreWhite, setScoreWhite] = useState("?");
+    const [scoreBlack, setScoreBlack] = useState("?");
+    const [teamWhite, setTeamWhite] = useState([]);
+    const [teamBlack, setTeamBlack] = useState([]);
+    const [goals, setGoals] = useState(Array<Goal>());
+    const [game, setGame] = useState();
+
+    const [qrCode, setQrCode] = useState("");
+    const [gameTime, setGameTime] = useState("");
+
+    function updateGameTime(startTime: number) {
+
+        let goalTime = Date.now();
+        let diff = goalTime - startTime;
+
+        let minutes = Math.floor(diff / 60000)
+        let minutesString = minutes.toString().length < 2 ? "0"+minutes.toString() : minutes.toString()
+        let seconds = Math.floor((diff / 1000) % 60)
+        let secondsString = seconds.toString().length < 2 ? "0"+seconds.toString() : seconds.toString()
+
+        setGameTime(`${minutesString}m ${secondsString}s`);
+    }
+
+    async function fetchData() {
+        const res = await fetch("https://tnl2api.trd.moe/p/monitor/1");
+        const json = await res.json();
+        
+        let teamWhite: Array<Player> = [];
+        let teamBlack: Array<Player> = [];
+        let game: Game;
+
+        if (json["Game"] === null) {
+            if (json["Pending"] !== null) {
+
+                let players: Array<Pending>
+                players = json["Pending"]
+                players.forEach((p) => {
+                    if (p.Pending.Team === 1) {
+                        teamBlack.push(p);
+                    } else if (p.Pending.Team === 2) {
+                        teamWhite.push(p);
+                    }
+                })
+            }
+
+        } else {
+            game = json.Game
+            updateGameTime(Date.parse(game.StartTime));
+
+            let players = game.Players;
+            players.forEach((p) => {
+                if (p.Team === 1) {
+                    teamBlack.push(p);
+                } else if (p.Team === 2) {
+                    teamWhite.push(p);
+                }
+            })
+        }
+
+        setTeamWhite(teamWhite);
+        setTeamBlack(teamBlack);
+        setGame(game);
+        setGoals(game?.Goals || []);
+        setScoreBlack(game?.ScoreBlack?.toString() || "?");
+        setScoreWhite(game?.ScoreWhite?.toString() || "?");

```

Comment:
```suggestion
        setGoals(game?.Goals ?? []);
        setScoreBlack(game?.ScoreBlack?.toString() ?? "?");
        setScoreWhite(game?.ScoreWhite?.toString() ?? "?");

```

I think `??` is more fitting since `||` would also trigger on empty arrays